data/mods/More_Survival_Tools/items.json

```diff
@@ -84,12 +84,12 @@
     "color": "light_gray",
     "name": "ember carrier (lit)",
     "name_plural": "ember carriers (lit)",
-    "description": "This is a small container for holding tinder, with holes punched in it to provide a controlled airflow.  An ember is smoldering inside it, and can be used to light a fire.",
+    "description": "This is a small container for holding tinder, with holes punched in it to provide a controlled airflow.  An ember is smoldering inside it, and can be used to start a campfire.",
     "price": 1000,
     "material": "iron",
     "flags": ["FIRE", "FLAMING", "REFILLABLE_LIGHTER"],
```

Comment:
Minor: None of those flags matter. Only "FLAMING" does anything and what it does is dealing fire damage on melee hit.